principles.md

```diff
@@ -0,0 +1,34 @@
+# Status Principles
+
+The goal of Status is widespread adoption of the decentralized web. Our challenge is achieving mass adoption while staying true to our principles outlined below.
+
+**Liberty**
+We believe in the sovereignty of individuals. As a platform that stands for the cause of personal liberty, we aim to maximize social, political, and economic freedoms.
+
+**Censorship resistance**
+We enable free flow of information. No content is under surveillance. We abide
+by the cryptoeconomic design principle of censorship resistance.
+
+**Security**
+We don't compromise on security when building features. We use state-of-the-art technologies, and research new security methods and technologies to make strong security guarantees.
+
+**Privacy**
+Privacy is the power to selectively reveal oneself to the world. For us, it's essential to protect privacy in both communications and transactions, as well as to preserve the right to total anonymity.
```

Comment:
Total Anonymity is an ideal we strive towards, however it's not feasible, Ethereum (and Status) is Pseudo-Anonymous

Reply:
pushed change